lib-find-meal/src/main/java/com/kinandcarta/lib/find/meal/adapter/MealsAdapter.kt

```diff
@@ -39,28 +39,28 @@ class MealsAdapter(var mealsList: List<Meal>) :
         private val portionsView = itemView.findViewById<TextView>(R.id.mealPortions)
         private val reserveButton = itemView.findViewById<Button>(R.id.reserveButton)
 
-        fun bindData(meal: Meal) {
-            nameView.text = meal.name
-            val tempString = if (meal.hot) "Hot" else "Cold"
-            val tempColor = if (meal.hot) R.color.colorHot else R.color.colorCold
+        fun bindData(item: MealWithDistance) {
+            nameView.text = item.meal.name
+            val tempString = if (item.meal.hot) "Hot" else "Cold"
+            val tempColor = if (item.meal.hot) R.color.colorHot else R.color.colorCold
             tempView.text = tempString
             tempView.setTextColor(ContextCompat.getColor(itemView.context, tempColor))
 
-            if (meal.info.isNullOrEmpty()) {
+            if (item.meal.info.isNullOrEmpty()) {
```

Comment:
I think there might be a linter warning on this line about using isNullOrEmpty when `info` isn't nullable

Reply:
It would be better if it was just `isEmpty() `?